Harvest.Web/Controllers/CropController.cs

```diff
@@ -142,5 +142,17 @@ public async Task<ActionResult> Delete(int id, Crop model)
                 return View();
             }
         }
+
+        // GET: CropController/Search?type=Row
+        public async Task<ActionResult> Search([FromQuery] string type, [FromQuery] string query = "")

```

Comment:
Can't have it here. This entire class if FM access.

Reply:
yea you might be able to override w/ a new `[Authorize]` attribute which changes the policy to just any authorized user and still keep it in the class, but yup it needs to be changed.

Reply:
Removed FM from class and applied to all methods but this one.